tests/SecurityDetails/SecurityDetailsConnector.test.ts

```diff
@@ -28,8 +28,8 @@ export async function securityDetailsLoad (
     Assert.deepStrictEqual(
         connector.table.getColumnNames(),
         [
-            'SecurityDetails_TrailingPerformance_TimePeriod',
-            'SecurityDetails_TrailingPerformance_Value'
+            'TrailingPerformance_TimePeriod',
+            'TrailingPerformance_Value'
```

Comment:
We should also update docs. That it is `TrailingPerformance_TimePeriod`, not `SecurityDetails_TrailingPerformance_TimePeriod`